lib/tasks/install.rake

```diff
@@ -0,0 +1,8 @@
+namespace :hyrax do
+  namespace :install do
+    desc 'Copy migrations from Hyrax to application'
+    task migrations: :environment do
```

Comment:
This task is added to replace the built-in task of the same name that was automatically added to Hyrax until the changes made in this PR. It should work in the same way. (The other task, provided by Railties, does not handle ERB evaluation.)